bash_scripts/install_robocup.sh

```diff
@@ -1,4 +1,4 @@
-sudo apt-get update \
+apt-get update \
```

Comment:
I assume the reason for this change is that the docker container doesn't need root access to run these commands. However, won't making this change cause problems for when people try to install robocup outside of docker?

Reply:
Good point. I will re-add sudo to the install. :+1: